packages/components/src/SVG/SVG.tsx

```diff
@@ -56,6 +56,7 @@ export const SVG = ({
   const accessibilityProps = {
     role,
     ["aria-hidden"]: isMeaningfulImg ? undefined : true,
+    ["aria-label"]: isMeaningfulImg ? title : undefined,
```

Comment:
I'm pretty sure `<title>` is meant to be the correct thing SVGs use as opposed to `aria-label`? Based on what I'm seeing, if `title` doesn't have a value, this won't help anyway.

Reply:
Ah my bad 😅